src/libraries/Common/src/Interop/Linux/os-release/Interop.OSReleaseFile.cs

```diff
@@ -0,0 +1,82 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+
+internal static partial class Interop
+{
+    // Parse information from '/etc/os-release'.
+    internal static class OSReleaseFile
+    {
+        private const string EtcOsReleasePath = "/etc/os-release";
```

Comment:
Should this fall back to [`/usr/lib/os-release`](https://www.freedesktop.org/software/systemd/man/os-release.html)?

Reply:
Do you have a particular distro or use-case in mind that needs this fall-back?

Reply:
I don't think a fallback is needed because we haven't needed one for the existing code that reads `/etc/os-release`.